cirq-superstaq/cirq_superstaq/compiler_output.py

```diff
@@ -99,10 +123,11 @@ def __init__(
         self.seq = seq
 
     def has_multiple_circuits(self) -> bool:
-        """Returns True if this object represents multiple circuits.
+        """Checks if an object has .circuits and .pulse_lists attributes. Otherwise, the
+        object represents a single circuit, and has .circuit and .pulse_list attributes.
```

Comment:
Shouldn't this be split into a single line + additional text after a newline?

Reply:
You're right, fixed now.